api/proto/info/info.proto

```diff
@@ -442,7 +442,7 @@ message StorageDiskState {
 enum StorageRaidType {
   STORAGE_RAID_TYPE_UNSPECIFIED = 0;
   STORAGE_RAID_TYPE_RAID0 = 1;             // RAID-0
-  STORAGE_RAID_TYPE_RAID1 = 2;        	   // Mirror 
+  STORAGE_RAID_TYPE_RAID1 = 2;             // Mirror
```

Comment:
Will proto updates produce generated code update? If so, nice to run `make proto&&make proto-vendor` and provide updates in separate commit.

Reply:
Ok, will do it...